tests/Sniffs/TestCase.php

```diff
@@ -157,14 +157,7 @@ protected function getSniffClassName(): string
 
 	protected function getSniffPath(): string
```

Comment:
@Majkl578 You're right. So new method `getSniffClassReflection()` with `static $reflectionClass` inside?